tests/System/integration/administrator/components/com_actionlogs/Actionlogs.cy.js

```diff
@@ -19,7 +19,7 @@ describe('Test in backend that the action logs', () => {
     cy.doAdministratorLogout();
     cy.doAdministratorLogin();
     cy.visit('/administrator/index.php?option=com_actionlogs&view=actionlogs');
-    cy.contains('User ci-admin logged in to admin');
+    cy.contains('User ${Cypress.env('username')} logged in to admin');
```

Comment:
```suggestion
    cy.contains(`User ${Cypress.env('username')} logged in to admin`);
```
I think you have to use that kind of tick marks for a string when variables shall be expanded in the string.

However, I'm not sure if an expression like `Cypress.env('username')` will also work here. We will see after you have applied this suggestion.